api/swagger.yaml

```diff
@@ -36,6 +36,20 @@ info:
     roll-back scenario's. A typical use case is when the originating system is not event based, and it has to communicate an entire "planning" at
     once. The benefits of using a `Bundle` are that you can save network traffic and that all entities and events are validated at the same time.
 
+    ## Changes since 4.1
+    - Rewrite of the [documentation on `Event`s](#tag/Event).
+    - Added `groupingId` to `Event`s.
```

Comment:
I think we also wanted something to model an order right? For example the order of shipments in a trip? Or maybe the order of trips in a larger trip?